estimate-web/src/components/UserStoryCommentList.less

```diff
@@ -0,0 +1,23 @@
+@import "../resources/Measurements.less";
+@import "../resources/Colors.less";
+
+.user-story-comments {
+    display: grid;
+    grid-template-columns: 50px auto;
+    grid-template-rows: auto;
+    margin-top: @base-margin;
+    gap: @margin-large 0;
+}
+
+.user-story-comment {
+    border: 1px solid @background-light;
```

Comment:
Might make sense to use `@base-border` or refactor this into an alternative reusable style